nuke-build/_build.csproj

```diff
@@ -0,0 +1,19 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>net9.0</TargetFramework>
```

Comment:
A bunch of the tooling is net9.0 and net9.0 is configured as the SDK. It doesn't impact the runtime used for the tests and the library itself still targets netstandard2.1 (and net462)

Reply:
Consider making that a comment in the XML